services/TransactionService/src/index.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Islandora\TransactionService;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use Islandora\Chullo\Chullo;
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Response;
+
+date_default_timezone_set('UTC');
+
+$app = new Application();
+
+$app['debug'] = true;
+
+$app['fedora'] = function () use ($app) {
+    $client = new Client(['base_uri' => 'http://127.0.0.1:8080/fcrepo/rest']);
```

Comment:
No reason in particular.  Doing it manually gives you a bit more control over the client, which in this case, is unneccessary.